src/store/modules/TsModsModule.ts

```diff
@@ -22,12 +28,40 @@ export const TsModsModule = {
     namespaced: true,
 
     state: (): State => ({
+        cache: new Map<string, CachedMod>(),
         deprecated: new Map<string, boolean>(),
         /*** All mods available through API for the current active game */
         mods: [],
     }),
 
     getters: <GetterTree<State, RootState>>{
+        /** Vue components mostly process mods in ManifestV2 format,
+         *  but sometimes need access to ThunderstoreMod format and
+         *  related data too. Since filtering the whole mods list every
+         *  time this happens slows down the LocalModList, cache the
+         *  data in a Map.
+         */
+        cachedMod: (state) => (mod: ManifestV2): CachedMod => {
+            const cacheKey = `${mod.getName()}-${mod.getVersionNumber()}`;
+
+            if (state.cache.get(cacheKey) === undefined) {
+                const tsMod = state.mods.find((m) => m.getFullName() === mod.getName());
```

Comment:
Should we not be able to query this from the IndexedDB now? Maybe you added that in a later PR already & I forgot

Reply:
Accessing IndexedDB is async and the sync Vuex getter is called by e.g. sync Vue computed properties. But I can look into it (not blocking this PR). At the very least the prewarming of cache could be done that way, although that would still require keeping the whole list in memory for other accessing code paths.